mm_cli.py

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env python3
+# CLI interface for mgamcmahon program
+
+import argparse
+import mcmahon
+import yaml
+
+parser = argparse.ArgumentParser(description='CLI interface for mgamcmahon program')
+
+parser.add_argument('--filename', '-f',
+                    action="store",
+                    default="tournament.yaml",
+                    help="Filename of data. Default is 'tournament.yaml'")
+parser.add_argument('--new-round', '-n',
```

Comment:
Personally I would prefer to avoid dashes for the actual commands (as opposed to flags like filename which merely modify said commands). So, for example, you might write "mm-cli new-round", or "mm-cli -f  /tmp/bs.yaml new-round". Basically, I see the command statements as arguments, not flags.